samples/testbot/deployment-service/README.md

```diff
@@ -0,0 +1 @@
+# deployment-service
```

Comment:
it might be good to clarify in here that the deployment service is a kind of "helper" to the testbot orchestration. I was unsure of the interaction when first navigating through the code